README.md

```diff
@@ -1,4 +1,34 @@
 # teslausb
+## Armbian orange pi zero 2 setup *tweaks
+
+apt pacakages:
+
+* xfsprogs
+* dos2unix
+* hping3
+* libxxhash0
+* autofs
+* python3-pip
+* nginx
+* fcgiwrap
+* libnginx-mod-http-fancyindex
+* libfuse-dev
+* exfatprogs
+* sntp
+
+first boot setup wifi with `nmtui`
+
+make a wpa_supplicant.conf at /etc/wpa_supplicant/wpa_supplicant.conf
+
+reboot
+
+to commit network changes
+
+now run the teslausb install
+
+```curl https://raw.githubusercontent.com/marcone/teslausb/main-dev/setup/generic/install.sh | bash```
+
+if you have the changes for createbackingfiles.sh in this repo your install should work and run fine, you may have to restart it after making the mountpoints for the shares/disk images
```

Comment:
The install scripts should take care of this already.

Reply:
Agreed, if the fixes to mkfs work with the armbian flavor of mkfs binaries that was the intent of this fork